.storybook/stories/useAnimations.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import { Vector3 } from 'three'
+import { GLTF } from 'three/examples/jsm/loaders/GLTFLoader'
+import { withKnobs, select, number } from '@storybook/addon-knobs'
+
+import { Setup } from '../Setup'
+
+import { useAnimations, useGLTF, useMatcapTexture } from '../../src'
+
+export default {
+  title: 'Abstractions/useAnimations',
+  component: useAnimations,
+  decorators: [(storyFn) => <Setup cameraPosition={new Vector3(0, 0, 3)}>{storyFn()}</Setup>, withKnobs],
+}
+
+type GLTFResult = GLTF & {
+  nodes: {
+    YB_Body: THREE.SkinnedMesh
+    YB_Joints: THREE.SkinnedMesh
+    mixamorigHips: THREE.Bone
+  }
+  materials: {
+    YB_Body: THREE.MeshStandardMaterial
+    YB_Joints: THREE.MeshStandardMaterial
+  }
+}
+
+type ActionName = 'Dance' | 'Idle' | 'Strut'
+type GLTFActions = Record<ActionName, THREE.AnimationAction>
+
+function YBot(props: JSX.IntrinsicElements['group']) {
+  const group = React.useRef<THREE.Group>()
+  const { nodes, animations } = useGLTF('ybot.glb') as GLTFResult
+  const { actions } = useAnimations(animations, group)
+
+  const [matcapBody] = useMatcapTexture('293534_B2BFC5_738289_8A9AA7', 1024)
+  const [matcapJoints] = useMatcapTexture('3A2412_A78B5F_705434_836C47', 1024)
+
+  // Storybook Knobs
+  const actionOptions = Object.keys(actions)
+  const selectedAction = select('Animation', actionOptions, actionOptions[2])
+  const blendDuration = number('Blend duration', 0.5, {
+    range: true,
+    min: 0,
+    max: 2,
+    step: 0.1,
+  })
+
+  React.useEffect(() => {
+    actions[selectedAction].reset().fadeIn(blendDuration).play()
+    return () => void actions[selectedAction].fadeOut(blendDuration)
+  }, [actions, selectedAction, blendDuration])
+
+  return (
+    <group ref={group} {...props} dispose={null}>
+      <group rotation={[Math.PI / 2, 0, 0]} scale={[0.01, 0.01, 0.01]}>
+        <primitive object={nodes.mixamorigHips} />
```

Comment:
This is really great to see & you've made the story look really lovely. I think if I was a first time user (which many are to `drei`, this story is a bit complicated to disect what part we're trying to show. 

I suggest that we abstract the model bits:
```tsx
<group rotation={[Math.PI / 2, 0, 0]} scale={[0.01, 0.01, 0.01]}>
    <primitive object={nodes.mixamorigHips} />
    <skinnedMesh geometry={nodes.YB_Body.geometry} skeleton={nodes.YB_Body.skeleton}>
      <meshMatcapMaterial attach="material" matcap={matcapBody} skinning={true} />
    </skinnedMesh>
    <skinnedMesh geometry={nodes.YB_Joints.geometry} skeleton={nodes.YB_Joints.skeleton}>
      <meshMatcapMaterial attach="material" matcap={matcapJoints} skinning={true} />
    </skinnedMesh>
  </group>
```

In to a new component called YBot & then have the old `YBot` could be called `YBotAnimationRig` or something of the sort. I'd just like to make it really simple and clear to someone at first glance, but I don't want to throw away the finesse you've added to the story 👍🏼

Reply:
No problem, thanks for the review. First time dealing with stories and to be honest I wasn't exactly sure what their use case was exactly, but that makes perfect sense. I'll try to emphasize the `useAnimations` a little more and abstract the model itself.

Reply:
I think the best view to have on this storybook is to show people the capabilities of the abstractions/helpers etc. and **how** to use them / write their code.

Reply:
But it doesn't need to be every feature displayed, it's a top level kind of thing.